das/aggregator_test.go

```diff
@@ -22,21 +22,26 @@ import (
 
 func TestDAS_BasicAggregationLocal(t *testing.T) {
 	numBackendDAS := 10
-	var backends []serviceDetails
+	var backends []ServiceDetails
 	for i := 0; i < numBackendDAS; i++ {
 		dbPath, err := ioutil.TempDir("/tmp", "das_test")
 		Require(t, err)
 		defer os.RemoveAll(dbPath)
 
-		signerMask := uint64(1 << i)
-		das, err := NewLocalDiskDataAvailabilityService(dbPath, signerMask)
+		config := LocalDiskDASConfig{
+			KeyDir:            dbPath,
```

Comment:
Thoughts on allowing public/private keys to optionally be specified through configuration (command line, env var, config file) to avoid having to maintain multiple files?

Reply:
I made this change in https://github.com/OffchainLabs/nitro/pull/589